src/components/reusable/booking/BookingItemCard.tsx

```diff
@@ -17,16 +19,21 @@ const BookingItemCard: React.FC<Booking> = ({
   currency,
   status,
 }) => {
+  const navigate = useNavigate();
+  const onClickNavigate = (id: string) => {
+    navigate(`/bookings/${id}`);
+  };
+
   return (
     <>
-      <Card>
+      <Card onClick={() => onClickNavigate(id)}>
```

Comment:
Actually there's no need to pass `id` here. You can get it from props
```
const onClickNavigate = useCallback(() => {
    navigate(`/bookings/${id}`);
  }, [id]);
```
And then just
```
<Card onClick={onClickNavigate}>
```